src/renderer/components/AircraftSection/VersionHistory.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styled from "styled-components";
+import { shell } from "electron";
+import { ModVersion } from "renderer/components/App";
+
+/**
+ * Container for versions
+ */
+export const Versions = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const GITHUB_RELEASE_BASE_URL = 'https://github.com/flybywiresim/a32nx/releases/tag/';
+
+type VersionProps = {
+    index: number,
+    className?: string,
+    version: ModVersion
+};
+
+const VersionBase: React.FC<VersionProps> = (props) => {
+    const ref = useRef<HTMLDivElement>();
+
+    const [versionContainerHeight, setVersionContainerHeight] = useState(0);
+
+    useEffect(() => {
+        const height = ref.current.clientHeight;
+
+        setVersionContainerHeight(height);
+    }, [ref]);
+
+    const openReleasePage = () => shell.openExternal(GITHUB_RELEASE_BASE_URL + props.version.title);
+
+    return (
+        <div ref={ref} className={props.className} onClick={openReleasePage}>
+            <VersionLine>
+                <path d={`m 10 ${props.index == 0 ? '15' : 0} l 0 ${versionContainerHeight}`} />
+
+                <circle cx={10} cy={15} r={5} />
+            </VersionLine>
+
+            <VersionTitle>{props.version.title}</VersionTitle>
+            <VersionDate>{props.version.date}</VersionDate>
+            <VersionType>{props.version.type} version</VersionType>
+        </div>
+    );
+};
+
+/**
+ * Visually displays a mod release
+ */
+export const Version = styled(VersionBase)`
+  display: grid;
+  grid-template-columns: [start] 1fr [middle] 6fr [end] 3fr;
+  grid-template-rows: [start] 30px [middle] auto;
+
+  letter-spacing: .04em !important;
+  
+  --timeline-node-color: ${props => props.index === 0 ? 'white' : '#757575'};
```

Comment:
**suggestion:** Ant design has already a timeline component if you are not aware of that
https://ant.design/components/timeline/